src/core_plugins/console/public/src/es.js

```diff
@@ -18,9 +18,20 @@ module.exports.send = function (method, path, data, server, disable_auth_alert)
   // delayed loading for circular references
   var settings = require("./settings");
 
+  let contentType;
+  if (data) {
+    try {
+      JSON.parse(data);
+      contentType = 'application/json';
```

Comment:
Another edge case is when users are not using the bulk API but submit slightly invalid JSON (like `{ name: 'val' }`). Elasticsearch will accept it, but a proxy or third party would likely fail to parse it.

Ultimately though elasticsearch doesn't consider the content-type at all, which is why we didn't bother with this to begin with. The only time we *should* set send `Content-Type: application/json` is when we think the body is in fact valid JSON, so this works for me.